docs/src/main/sphinx/functions.rst

```diff
@@ -2,13 +2,15 @@
 Functions and operators
 ***********************
 
-This chapter describes the SQL functions and operators supported by Trino. They
-allow you to implement complex functionality and behavior of the SQL executed by
-Trino operating on the underlying data sources.
+This chapter describes the built-in SQL functions and operators supported by
+Trino. They allow you to implement complex functionality and behavior of the SQL
+executed by Trino operating on the underlying data sources.
 
 If you are looking for a specific function or operator, see the :doc:`full
 alphabetical list</functions/list>` or the :doc:`full list by
-topic</functions/list-by-topic>`.
+topic</functions/list-by-topic>`. Using :doc:`SHOW FUNCTIONS
+</sql/show-functions>` returns a list of all available functions, including
+custom functions, with all supported arguments and a short description.
```

Comment:
Confirmed:

     Function     | Return Type | Argument Types | Function Type | Deterministic |               Description
------------------+-------------+----------------+---------------+---------------+-----------------------------------------
 cus_getgroups    | varchar     |                | scalar        | true          | Returns the current session's groups
 cus_getprincipal | varchar     |                | scalar        | true          | Returns the current session's principal
 cus_getuser      | varchar     |                | scalar        | true          | Returns the current session's user